### PR TITLE
update Russian translation

### PR DESCRIPTION
**Pull request by openmindead**
Hello.
Here is an update of Russian translation for systemd messages.
Some phrases may sound weird because there's no a good translation for things like "home area", but I did my best to make them as clear as possible.

BR,
Vladimir

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -696,38 +946,60 @@ msgstr ""
...
+#~ "to inhibit it."
+#~ msgstr ""
+#~ "Чтобы остановить систему, несмотря на то, что приложение запросило "
+#~ "блокировку выключения, необходимо пройти аутентификацию."
+
+#~ msgid "Authentication is required to kill '$(unit)'."
+#~ msgstr "Чтобы убить юнит «$(unit)», необходимо пройти аутентификацию."
 
 #~ msgid "Press Ctrl+C to cancel all filesystem checks in progress"
 #~ msgstr ""
```
Comment (yuwata) [inline po/ru.po]: Please drop unused translations.

Comment (openmindead) [reply po/ru.po]: Oops, ごめんなさい。

**Comment by openmindead** [comment]
Done!

**Comment by yuwata** [comment]
LGTM.

**Pull request merged**